commit 12c9c51dc2603ffb09dffdf051116bac7e603dde (refs/changes/72/2272/1)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-07-25T00:50:27+00:00 (2 years, 2 months ago)
    
    NAPI-409 allow listing of network pools by UUID prefix (fix make check)

diff --git a/lib/util/validate.js b/lib/util/validate.js
index d7e8af0..e65d94a 100644
--- a/lib/util/validate.js
+++ b/lib/util/validate.js
@@ -30,14 +30,8 @@ var warden = require('restify-warden');
 // --- Globals
 
 
-
 var INTERFACE_NAME_RE = /[a-zA-Z0-9_]{0,31}/;
 var INTERFACE_NUM_RE = /[0-9]+$/;
-var STR_RE = /\s/g;
-
-
-// --- Internal helpers
-
 
 
 // --- Exports
diff --git a/test/integration/network-pools.test.js b/test/integration/network-pools.test.js
index b6f3329..dd5317a 100644
--- a/test/integration/network-pools.test.js
+++ b/test/integration/network-pools.test.js
@@ -157,7 +157,7 @@ test('GET /network_pools?uuid=badcafe*', function (t) {
 
 test('GET /network_pools?uuid=$existing_prefix', function (t) {
     var uuid = state.pools[0].uuid;
-    var prefix = uuid.substring(0, 8) + "*";
+    var prefix = uuid.substring(0, 8) + '*';
     /* Expect empty list */
     mod_pool.list(t, {
         params: {uuid: prefix},
diff --git a/test/integration/networks.test.js b/test/integration/networks.test.js
index 1c71733..21c3677 100644
--- a/test/integration/networks.test.js
+++ b/test/integration/networks.test.js
@@ -276,7 +276,7 @@ test('GET /networks (filtered)', function (t) {
 
 test('GET /networks?uuid=$existing_prefix', function (t) {
     var uuid = state.network.uuid;
-    var prefix = uuid.substring(0, 8) + "*";
+    var prefix = uuid.substring(0, 8) + '*';
     mod_net.list(t, {
         params: {uuid: prefix},
         present: [ state.network ]
